ocs_ci/helpers/dr_helpers.py

```diff
@@ -1661,3 +1661,57 @@ def generate_kubeobject_capture_interval():
         return 5
     else:
         return capture_interval
+
+
+def disable_dr_rdr():
```

Comment:
the function has similarities to https://github.com/red-hat-storage/ocs-ci/blob/7686692d42a47d9624813a2df312011b8965a97c/ocs_ci/helpers/dr_helpers.py#L1458. 
Can we merge them?

Reply:
> the function has similarities to
> 
> https://github.com/red-hat-storage/ocs-ci/blob/7686692d42a47d9624813a2df312011b8965a97c/ocs_ci/helpers/dr_helpers.py#L1458
> 
> .
> Can we merge them?

@ebenahar This function has been written for cluster replacement scenario in mdr, which altogether has longer steps while disabling DR. Moreover, for rdr, we have additional step of adding new annotations to drpc before deleting it. If I were to use this function, I would have to introduce two new flags. one flag for rdr - as, in mdr, they used `run_cmd_verify_cli_output`
Which we cannot use for rdr because we need to validate the o/p from _stderr_ but `run_cmd_verify_output` validates o/p from _stdout_., for this purpose, I wrote another small new function `verify_drpc_deletion`, and also for additional step (adding annotation) in rdr, second flag is for cluster replacement - to have additional steps different from simple disable DR. When I started my automation, the PR that contained function `disable_dr_from_app ` was in the state of inactivity so I could not rely on it for simple disable DR functionality. We can definitely consider using this existing function while automating cluster replacement.